### PR TITLE
Minor formatting changes

### PR DESCRIPTION
**Pull request by dawud**

### CODE REVIEW

File: docs/6-cluster-files.md [hunk 1]
```diff
@@ -1,23 +1,31 @@
 # Cluster files
-openshift-install requires some files (pull secret and install-config) and creates some other assets (manifests, ignition files, logs, etc.). In order to have a proper directory structure, they will be stored in ~/ocp-clusters/<cluster-name>:
 
-```
+`openshift-install` requires some files (pull secret and install-config) and
+creates some other assets (manifests, ignition files, logs, etc.). In order to
+have a proper directory structure, they will be stored in
+~/ocp-clusters/<cluster-name>:
```
Comment (e-minguez) [inline docs/6-cluster-files.md]: Do you mind to backquote this as well?

**Review by e-minguez — APPROVED** [review]
Thank you for the PR @dawud ! I've just added a nit. LGTM in any case :)

File: docs/6-cluster-files.md (hunk 1 above)
Comment (dawud) [reply docs/6-cluster-files.md]: Should be fixed now.

**Comment by e-minguez** [comment]
Merged, thanks!!!

**Pull request merged**